### ISSUE TITLE
Lack of contrast of greyed labels

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#574]
We should reach 4.5 to 1 contrast (WCAG 2.0).
First, change the light grey of inactive buttons in the UI for a slightly darker grey which reaches 4.5 to 1. 
After some tests, we’ll use #767676 as “light” grey. It passes WCAG AA, plus AAA for large text. A high contrast theme will later complete the capabilities of the app.
Checked using color contrast analyser https://developer.paciellogroup.com/resources/contrastanalyser/

**Comment by Scarsniik** [comment]
fixed in #583
